variables.tf

```
@@ -0,0 +1,47 @@
+variable "service_name" {
+  type        = string
+  description = "Name of service."
+  default     = ""
+}
+
+variable "service_image" {
+  type        = string
+  description = "Service container image url."
+  default     = ""
+}
+
+variable "service_memory" {
+  type        = string
+  description = "Allocated memory to pods."
+  default     = "512Mi"
+}
+
+variable "service_cpu" {
+  type        = string
+  description = "Allocated cpu to pods."
+  default     = "0.5"
+}
+
+variable "pods_replica_count" {
+  type        = number
+  description = "The number of desired pods replicas."
+  default     = 1
+}
+
+variable "container_port" {
```

```suggestion
variable "service_port" {
```

Let's stay consistent with fargate service.

---

Taken care.